test/image/mocks/subtitle.json

```diff
@@ -0,0 +1,33 @@
+{
+    "data": [
+        {
+            "x": [1, 2, 3, 4],
+            "y": [10, 12, 11, 13],
+            "type": "scatter"
+        }
+    ],
+    "layout": {
+        "title": {
+            "text": "A simple plot",
+            "subtitle": {
+                "text": "With a subtitle"
+            }
+        },
+        "xaxis": {
+            "title": {
+                "text": "x-axis",
+                "subtitle": {
```

Comment:
Adding `subtitle` to cartesian axes, radial axes, colorbars, etc. might be out of the scope of this PR.
I think you would delete this file?

Reply:
whoops, yes, I guess this is an old version that snuck in here after I changed the file name. Will remove!